common/traits/mergeable/src/lib.rs

```diff
@@ -6,14 +6,27 @@ pub static CANNOT_MERGE_ERR_MSG: &[u8] = b"Cannot merge";
 
```

Comment:
Maybe an ERR_MSG file would be useful?

Reply:
Eh, if it was a smart contract, maybe, but since this is a standalone module, I'd rather each keep their own error message, then a big file with all the errors combined.